main/webapp/modules/core/scripts/reconciliation/recon-dialog.js

```diff
@@ -31,11 +31,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
  */
 
-function ReconDialog(column, types) {
+function ReconDialog(column,previousServiceName) {
   this._column = column;
   this._serviceRecords = [];
   this._selectedServiceRecordIndex = -1;
-
+  this._record=null;
+  this.previousRecord=previousServiceName;
```

Comment:
For consistency with the rest of the style, I think it would be better to have space characters around the equal signs.